modules/rating/src/main/Glicko.scala

```diff
@@ -42,15 +47,22 @@ case object Glicko {
 
   val minRating = 800
 
-  val default = Glicko(1500d, 350d, 0.06d)
-
-  val defaultIntRating = default.rating.toInt
+  val maxDeviation = 350d
+  val provisionalDeviation = 110d
 
-  val provisionalDeviation = 110
+  // amount to age the RD over a rating period
+  // 60 months (rating periods) would need to pass before a rating for a typical
+  // player becomes as uncertain as that of an unrated player
+  val c = sqrt(pow(maxDeviation, 2) - pow(provisionalDeviation / 2, 2) / 60d)
+  val daysPerPeriod = 30d
+  def periods(latest: Option[DateTime], date: DateTime): Double = Days.daysBetween(date, latest | date).getDays / daysPerPeriod
```

Comment:
general perf: float math if possible and avoid divides when possible (in this case multiply by reciprocal)